src/app/pages/fund/components/zero-percent-fees-badge.tsx

```diff
@@ -0,0 +1,30 @@
+import { FiStar } from 'react-icons/fi';
+
+import { Box, color, Stack } from '@stacks/ui';
+import { Caption } from '@app/components/typography';
+
+export function ZeroPercentFeesBadge() {
+  return (
+    <Box
+      border="1px solid"
+      borderColor="#FFE0C2"
+      borderRadius="25px"
+      color={color('text-caption')}
+      height="24px"
+      width="80px"
```

Comment:
same thoughts on width. Is there any duplicate code we could refactor into a badge maker component?